packages/pwa/app/utils/locale.test.js

```diff
@@ -17,17 +17,14 @@ import {SUPPORTED_LOCALES, DEFAULT_LOCALE} from '../constants'
 
 const supportedLocales = getSupportedLocalesIds()
 const nonSupportedLocale = 'nl-NL'
-// Make sure this supported locale is not the default locale.
-// Otherwise, our code would fall back to default and incorrectly pass the tests
-const supportedLocale = supportedLocales[1]
+
+const supportedLocale = supportedLocales[0]
```

Comment:
🤔 hmm, this test file assumes that it's working with a multi-locale setup.

Reply:
I see that we have updated the tests such that they would pass for both the `pwa` and the generated pwa project. But since the two have deviated from each other (one is a multi-locale site, while the other one is now a single locale), the tests can be more tricky to write and may not be clear to new developers.

How about we consider _not_ running the tests for the generated project? https://github.com/SalesforceCommerceCloud/pwa-kit/blob/0073095f7ca08ac03205943b0934558e79c2cfe7/.circleci/config.yml#L243-L244

Reply:
The generated project is our final most crucial product shipped to customers. 
I think it is vital for us to run tests on the code that we hand to customers to make sure that the code they use as starting point works as expected.

You're right that writing tests that work for a single-locale and multi-locale setup can be harder to write. Let me see what I can do on that test file to simplify but still cover both single-locale and multi-locale scenarios.